BankSDK/GiniBankSDKExample/Tests/FileLoader.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  FileLoader.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+
+struct FileLoader {
```

Comment:
I will check! Thanks for telling me!

Reply:
I checked, and yes we can use it from `GiniBankAPiLibrary`  but personally I don't like that method since is using force unwrap for the `fileURLPath` and the `data` extracted from that file wich may cause some crashes if the file can be read from the disk or data can not be extracted from that file.

I prefer to really avoid force unwrapping or try to reduce this if possible even if I duplicate, right now, the functionality of the method. Of course, I can change the implementation of that method but doesn't make sense to modify how is used in other places.

Also, this kind of method is present in other places as well like: TransferSummaryIntegrationTest in GiniBankAPILibraryPinningExampleTests and not only.

Reply:
Also to be able to use that method I need to make it public to be able to use it outside of `GiniBankAPiLibrary`. 
In my opinion, we really need to gather the common code in a different module/package/something and then we can reuse some utils method that we have duplicated in multiple places now.

Reply:
we agreed that create a shared package in the future